app/service/rest.py

```diff
@@ -210,6 +214,98 @@ def get_service_notification_statistics(service_id):
     )
 
 
+@service_blueprint.route("/<uuid:service_id>/statistics/<string:start>/<int:days>")
+def get_service_notification_statistics_by_day(service_id, start, days):
+    return jsonify(
+        data=get_service_statistics_for_specific_days(service_id, start, int(days))
+    )
+
+
+def get_service_statistics_for_specific_days(service_id, start, days=1):
+    start_date = datetime.strptime(start, "%Y-%m-%d").date()
+
+    def generate_date_range(start_date, days):
+        current_date = start_date
+        end_date = start_date - timedelta(days=days)
+        while current_date > end_date:
```

Comment:
Why is end date < start date and then current date counts backward until it matches start date?

Why not have end date == start + days and have current date count upwards?

Reply:
Some functions start from a date and move backward like "show be the last 7 days", in that case you're given a start date to go backward from. Others, like all days in a month, start from the beginning date of the month and count forward until the end of the month.